README.md

```diff
@@ -12,15 +12,10 @@ This repository contains:
 
 - [Housing characteristics of the U.S. residential building stock](https://github.com/NREL/resstock/tree/main/project_national/housing_characteristics), in the form of conditional probability distributions stored as tab-separated value (.tsv) files. Comments at the bottom of each file document data sources and assumptions for each.
 - [A library of housing characteristic "options"](https://github.com/NREL/resstock/blob/main/resources/options_lookup.tsv) that translate high-level characteristic parameters into arguments for OpenStudio measures, and which are referenced by the housing characteristic .tsv files and building energy upgrades defined in project definition files
-- Project definition files:
-  - v2.3.0 and later: [buildstockbatch YML files openable in any text editor](https://github.com/NREL/resstock/blob/main/project_national/national_baseline.yml)
-  - v2.2.5 and prior: [Project folder openable in PAT](https://github.com/NREL/resstock/tree/v2.2.5/project_singlefamilydetached)
-- Unit-level OpenStudio Measures for automatically constructing OpenStudio Models of each representative dwelling unit model:
-  - v3.0.0 and later: [OpenStudio-HPXML Measures](https://github.com/NREL/resstock/tree/main/resources/hpxml-measures)
-  - v2.5.0 and prior: [OpenStudio Measures](https://github.com/NREL/resstock/tree/v2.5.0/resources/measures)
+- Project definition [buildstockbatch YML files](https://github.com/NREL/resstock/blob/main/project_national/national_baseline.yml) openable in any text editor for v2.3.0 and later
+- Unit-level [OpenStudio-HPXML Measures](https://github.com/NREL/resstock/tree/main/resources/hpxml-measures) (v3.0.0 and later) for automatically constructing individual representative dwelling unit OpenStudio Models using [OpenStudio-HPXML](https://github.com/NREL/OpenStudio-HPXML)
 - [Higher-level OpenStudio Measures](https://github.com/NREL/resstock/tree/main/measures) for controlling simulation inputs and outputs
 
 This repository does not contain software for running ResStock simulations, which can be found as follows:
```

Comment:
I suppose this statement is technically not true anymore (we have run_analysis.rb).

Reply:
I changed the language to include `run_analysis.rb`.